Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -1700,7 +1715,7 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual("global_nav", dataLakeClient.properties.last?["context_location"] as? String)
     XCTAssertEqual("global_nav", segmentClient.properties.last?["context_location"] as? String)
 
-    ksrAnalytics.trackProjectSearchView()
+    ksrAnalytics.trackProjectSearchView(params: .defaults, results: 0)
```

Comment:
I would omit the results parameter (it has a default argument of nil) since this method is only checking certain context properties.